src/main/java/com/mraof/minestuck/client/gui/playerStats/EcheladderScreen.java

```diff
@@ -179,21 +186,62 @@ private void drawLadder(PoseStack poseStack, int currentRung, boolean showLastRu
 				textColor = rand.nextInt(0xFFFFFF);
 				if(textColors.length > rung)
 					textColor = textColors[rung];
-				fill(poseStack, xOffset + 90, y, xOffset + 236, y + 12, backgrounds.length > rung ? backgrounds[rung] : (textColor^0xFFFFFFFF));
+				fill(poseStack, xOffset + 90, y, xOffset + 236, y + 12, backgrounds.length > rung ? backgrounds[rung] : (textColor ^ 0xFFFFFFFF));
 			} else if(rung == currentRung + 1 && animationCycle == 0)
 			{
 				int bg = ~rand.nextInt(0xFFFFFF);
 				if(backgrounds.length > rung)
 					bg = backgrounds[rung];
 				else if(textColors.length > rung)
 					bg = ~textColors[rung];
-				fill(poseStack, xOffset + 90, y + 10, xOffset + 90 + (int)(146* ClientPlayerData.getRungProgress()), y + 12, bg);
+				fill(poseStack, xOffset + 90, y + 10, xOffset + 90 + (int) (146 * ClientPlayerData.getRungProgress()), y + 12, bg);
```

Comment:
I don't think the first extra space on this line is doing anyone any favors, and it can be removed.
That said, it's not like it's really hurting anything, either.

Reply:
We use a formatting system for standardizing line spacing, brackets, etc. In IntelliJ you can go to the Code tab or you can press Ctrl + Alt + L while in a file to Reformat Code. Its definetely not a big deal but considering it takes a few seconds per file would you please do so

Reply:
done on all files. using this to test resolve feture